### ISSUE TITLE
Add explanation for conceptual decisions

### ISSUE THREAD

**Issue by 1kastner** [theia-ide/theia-apps#340]
Is there an explanation for what has been included into which docker image? A maybe stupid question for people who have deeper insights in the project: Why don't I have an extension manager on the left side when starting the IDE? I wish the delivered version could be closer to VS code. This would make the transition much easier. Maybe there is a better place for a discussion like this. If so, could a link be added to the README.md? That'd be great! Thanks so much for all the contributions!

**Comment by vince-fugnitto** [comment]
@1kastner the `extensions` view is not yet part of the images since it is a fairly [new extension](https://github.com/eclipse-theia/theia/pull/6655). Note that due to license issues and terms of use, any non-official vscode product is forbidden to use the marketplace, that is why we have created the open-source alternative [open-vsx](https://open-vsx.org/). Until this was created, an `extensions` view was not possible, but it is now implemented :) We have a pull-request which adds the view to most images:
- https://github.com/theia-ide/theia-apps/pull/330

If you have additional feedback of what might be nice to include in the `readme`, I'd suggest giving some feedback to the following pull-request #279 which updates the `readme`.
